tern/analyze/default/container/image.py

```diff
@@ -73,3 +77,25 @@ def analyze(image_obj, options):
         passthrough.run_extension(image_obj, options.extend, options.redo)
     else:
         default_analyze(image_obj, options)
+
+
+def common_container_procedure(image, options, image_file=None):
```

Comment:
Small nit: What do you think about renaming this function to something a little more intuitive/descriptive? Maybe `common_image_setup` or `prepare_for_analysis`? @nishakm do you have a preference?

Reply:
I actually asked @m1-key to call it "common_container_procedure". It's not an image setup thing but a list of steps in common with analyzing a given image and a given dockerfile. I am open to naming it something else that communicates this function/subroutine.

Reply:
Ah, ok.. sorry @m1-key for the confusion!! I guess the "procedure" is just kind of throwing me off when paired with the function description that says """Setup the image object and anything else for analysis""". What about `common_analysis_procedure`?